fix hardcoding of 'g++' and '-O3' in Makefile, pick up $CXX and $CXXFLAGS instead
author: Kenneth Hoste (HPC-UGent)
diff --git a/v1.12-r6/Release/makefile.orig b/v1.12-r6/Release/makefile
index 3d8e958..6641ddf 100644
--- a/v1.12-r6/Release/makefile.orig
+++ b/v1.12-r6/Release/makefile
@@ -43,7 +43,7 @@ all: GapCloser
 GapCloser: $(OBJS) $(USER_OBJS)
 	@echo 'Building target: $@'
 	@echo 'Invoking: Cygwin C++ Linker'
-	g++  -o "GapCloser" -O3 $(OBJS) $(USER_OBJS) $(LIBS)
+	$(CXX)  -o "GapCloser" $(CXXFLAGS) $(OBJS) $(USER_OBJS) $(LIBS)
 	@echo 'Finished building target: $@'
 	@echo ' '
 
diff --git a/v1.12-r6/Release/subdir.mk.orig b/v1.12-r6/Release/subdir.mk
index 0261d4e..7808d75 100644
--- a/v1.12-r6/Release/subdir.mk.orig
+++ b/v1.12-r6/Release/subdir.mk
@@ -17,7 +17,7 @@ CPP_DEPS += \
 %.o: ../%.cpp
 	@echo 'Building file: $<'
 	@echo 'Invoking: Cygwin C++ Compiler'
-	g++ -I../ -I../../phoenix -I../../datastructure -O3 -Wall -c -fmessage-length=0 -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@:%.o=%.d)" -o"$@" "$<"
+	$(CXX) -I../ -I../../phoenix -I../../datastructure $(CXXFLAGS) -Wall -c -fmessage-length=0 -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@:%.o=%.d)" -o"$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
 
